metals/src/main/scala/scala/meta/internal/metals/MetalsEnrichments.scala

```diff
@@ -411,6 +411,13 @@ object MetalsEnrichments
 
     def toAbsolutePath: AbsolutePath =
       AbsolutePath(Paths.get(URI.create(value.stripPrefix("metals:")))).dealias
+
+    def findAll(pattern: String): Iterator[Int] = {
```

Comment:
Did you try using `findAllMatchIn`?

<img width="557" alt="Screenshot 2019-12-15 at 11 33 17" src="https://user-images.githubusercontent.com/1408093/70862042-c4dd1000-1f2e-11ea-8b32-23041ae290be.png">